Object Store/func.py

```diff
@@ -1,67 +1,85 @@
 import io
-import oci
-import re
 import os
+import gzip
 import json
-import sys
-import requests
 import logging
-import time
-import gzip
 
-from fdk import response
+import oci
+import requests
 
 
-def handler(ctx, data: io.BytesIO=None):
+def handler(ctx, data: io.BytesIO = None) -> None:
     try:
         body = json.loads(data.getvalue())
-        data = body.get("data", {})
-        additional_details = data.get("additionalDetails", {})
-        namespace = additional_details.get("namespace")
-        bucket = additional_details.get("bucketName")
-        obj = data.get("resourceName")
-        eventtime = body.get("eventTime")
+    except (Exception, ValueError) as ex:
```

Comment:
I believe [Exception](https://docs.python.org/3/library/exceptions.html#Exception) is a base class ontop of which other built in exceptions are created. This will probably catch all exceptions. If this is desired, then there is no need for ValueError here

Reply:
Agreed it's not functional, was more marking that it was known that the `ValueError` would pop up. But can update accordingly minor detail.